wsmaster/codenvy-sql-schema/src/main/resources/codenvy-schema/5.0.0-M8/1.1__init.sql

```diff
@@ -0,0 +1,362 @@
+
+
+-- Stack permissions -----------------------------------------------------------
+CREATE TABLE stackpermissions (
+    id          VARCHAR(255)         NOT NULL,
+    stackid     VARCHAR(255),
+    userid      VARCHAR(255),
+
+    PRIMARY KEY (id)
+);
+-- indexes
+CREATE UNIQUE INDEX index_stackpermissions_userid_stackid ON stackpermissions (userid, stackid);
+CREATE INDEX index_stackpermissions_stackid ON stackpermissions (stackid);
+-- constraints
+ALTER TABLE stackpermissions ADD CONSTRAINT fk_stackpermissions_userid FOREIGN KEY (userid) REFERENCES usr (id);
+ALTER TABLE stackpermissions ADD CONSTRAINT fk_stackpermissions_stackid FOREIGN KEY (stackid) REFERENCES stack (id);
+--------------------------------------------------------------------------------
+
+
+-- Stack permissions actions ---------------------------------------------------
+CREATE TABLE stackpermissions_actions (
+    stackpermissions_id     VARCHAR(255),
+    actions                 VARCHAR(255)
+);
+-- indexes
+CREATE INDEX index_stackpermissions_actions_actions ON stackpermissions_actions (actions);
+-- constraints
+ALTER TABLE stackpermissions_actions ADD CONSTRAINT fk_stackpermissions_actions_stackpermissions_id FOREIGN KEY (stackpermissions_id) REFERENCES stackpermissions (id);
+--------------------------------------------------------------------------------
+
+
+-- Recipe permissions ----------------------------------------------------------
+CREATE TABLE recipepermissions (
+    id          VARCHAR(255)         NOT NULL,
+    recipeid    VARCHAR(255),
+    userid      VARCHAR(255),
+
+    PRIMARY KEY (id)
+);
+-- indexes
+CREATE UNIQUE INDEX index_recipepermissions_userid_recipeid ON recipepermissions (userid, recipeid);
+CREATE INDEX index_recipepermissions_recipeid ON recipepermissions (recipeid);
+-- constraints
+ALTER TABLE recipepermissions ADD CONSTRAINT fk_recipepermissions_userid FOREIGN KEY (userid) REFERENCES usr (id);
+ALTER TABLE recipepermissions ADD CONSTRAINT fk_recipepermissions_recipeid FOREIGN KEY (recipeid) REFERENCES recipe (id);
+--------------------------------------------------------------------------------
+
+
+-- Recipe permissions actions --------------------------------------------------
+create table recipepermissions_actions (
+    recipepermissions_id    varchar(255),
+    actions                 VARCHAR(255)
+);
+-- indexes
+create index index_recipepermissions_actions_actions on recipepermissions_actions (actions);
+-- constraints
+ALTER TABLE recipepermissions_actions ADD CONSTRAINT fk_recipepermissions_actions_recipepermissions_id FOREIGN KEY (recipepermissions_id) REFERENCES recipepermissions (id);
+--------------------------------------------------------------------------------
+
+
+-- Organization ----------------------------------------------------------------
+CREATE TABLE organization (
+    id          VARCHAR(255)         NOT NULL,
+    parent      VARCHAR(255),
+    account_id  VARCHAR(255)         NOT NULL,
```

Comment:
Should we not use `_` symbol for fields names? Looks like you do it in both ways: with _ for [workers](https://github.com/codenvy/codenvy/pull/1161/files#diff-6cd6eca6d13e6148e2265297d10383a3R216) and without `_` for [member](https://github.com/codenvy/codenvy/pull/1161/files#diff-6cd6eca6d13e6148e2265297d10383a3R303)